### PR TITLE
Update intro/index.html with meaning of acronyms

### PR DESCRIPTION
**Pull request by LLyaudet**
Hello,
I just wanted to help the readability of this nice introduction,
by giving the meaning of acronyms for non-specialists like me :).
Best regards,
     Laurent Lyaudet

### CODE REVIEW

File: intro/index.html [hunk 1]
```diff
@@ -64,7 +64,7 @@
 3. [**Warmup - expressions**](#expressions) - a quick tour of some fundamental expression building blocks
 4. [**Inside an attention head**](#attnhead) - an in-depth look at the structure, values and computation behavior of a couple of attention heads from GPT2 via[ NanoGPT](https://github.com/karpathy/nanoGPT)
 5. [**Parallelizing attention**](#attnpar) - visualizing attention head parallelization with examples from the recent[ Blockwise Parallel Transformer](https://arxiv.org/pdf/2305.19370.pdf) paper
-6. [**Sizes in an attention layer**](#attnsizes) - what do the MHA and FFA halves of an attention layer look like together, when we visualize a whole layer as a single structure? How does the picture change during autoregressive decoding?
+6. [**Sizes in an attention layer**](#attnsizes) - what do the MHA (Multi-Headed Attention) and FFA (Firefly Algorithm) halves of an attention layer look like together, when we visualize a whole layer as a single structure? How does the picture change during autoregressive decoding?
```
Comment (bhosmer) [inline intro/index.html:67]: Ah you've actually found a typo, this acronym should be `FFN` (as below). Would you mind making the change to "FFN (Feed Forward Network)" here?

**Review by bhosmer — CHANGES_REQUESTED** [review]
Hi @LLyaudet - these are very useful additions, thank you! As noted inline, there's an error in the original text - if you could update your change to fix my mistake, I'll go ahead and merge. Thanks again!

File: intro/index.html (hunk 1 above)
Comment (LLyaudet) [reply intro/index.html]: No problem, I just did :)

**Comment by LLyaudet** [comment]
Hello @bhosmer :)
You're welcome !
I corrected the typo FFA -> FFN as you requested :)

**Review by bhosmer — APPROVED** [review]
Awesome, thanks Laurent!

**Pull request merged**